interface/modules/custom_modules/oe-module-claimrev-connect/table.sql

```diff
@@ -4,16 +4,23 @@ CREATE TABLE IF NOT EXISTS `mod_claimrev_eligibility`(
     ,`pid` bigint(20)
     ,`payer_responsibility` varchar(2)
     ,`request_json` TEXT
-    ,`response_json` TEXT
-	,`eligibility_json` TEXT
-	,`individual_json` TEXT
+    ,`response_json` LONGTEXT
+	,`eligibility_json` LONGTEXT
+	,`individual_json` LONGTEXT
     ,`response_message` TEXT
     ,`status` varchar(25)
     ,`last_checked` datetime default NULL
     ,`create_date` datetime default NULL
-    ,`raw271` MEDIUMTEXT
-);
+    ,`raw271` LONGTEXT
+); 
 
+ALTER TABLE `mod_claimrev_eligibility` 
+CHANGE COLUMN `response_json` `response_json` LONGTEXT NULL DEFAULT NULL ,
+CHANGE COLUMN `eligibility_json` `eligibility_json` LONGTEXT NULL DEFAULT NULL ,
+CHANGE COLUMN `individual_json` `individual_json` LONGTEXT NULL DEFAULT NULL ,
+CHANGE COLUMN `raw271` `raw271` LONGTEXT NULL DEFAULT NULL ;
```

Comment:
should be in a if (there is a if mechanism to avoid breaking things in database) and should should not mention defaults for longtext. here is what would change above to:
```
#IfNotColumnType mod_claimrev_eligibility response_json LONGTEXT
ALTER TABLE `mod_claimrev_eligibility` CHANGE `response_json` `response_json` LONGTEXT;
#EndIf

#IfNotColumnType mod_claimrev_eligibility eligibility_json LONGTEXT
ALTER TABLE `mod_claimrev_eligibility` CHANGE `eligibility_json` `eligibility_json` LONGTEXT;
#EndIf

#IfNotColumnType mod_claimrev_eligibility individual_json LONGTEXT
ALTER TABLE `mod_claimrev_eligibility` CHANGE `individual_json` `individual_json` LONGTEXT;
#EndIf

#IfNotColumnType mod_claimrev_eligibility raw271 LONGTEXT
ALTER TABLE `mod_claimrev_eligibility` CHANGE `raw271` `raw271` LONGTEXT;
#EndIf
```

Reply:
made those changes. I was actually trying to figure out that if logic but couldn't.